.github/workflows/test.yaml

```diff
@@ -117,6 +117,7 @@ jobs:
         if:
           matrix.groonga-version == 'main'
         run: |
+          sudo gem install rubygems-requirements-system
```

Comment:
I think we could remove the step for enabling the Apache Arrow repository.

https://github.com/groonga/groonga-nginx/blob/57cb60f0dd64c8980bb9d1c185421afebaa4c14a/.github/workflows/test.yaml#L105-L115

https://github.com/apache/arrow/blob/main/ruby/red-arrow/red-arrow.gemspec